client/client.rs

```diff
@@ -277,6 +321,9 @@ impl Client {
                 all,
                 children,
             } => {
+                if self.settings.client.print_remove_warnings {
```

Comment:
One last thing :D

`print_remove_warnings` is probably not the best name for this feature.
It's not only a print, but also a question and it's shown on `kill` and `remove`.

I'm struggling to come up with a good name as well, though. Stuff like that is always super hard.
I came up with something like `show_confirmation_questions`.

That should hopefully clarify what this option is used for, without having to read the `README`.

Sorry for only mentioning this now. It didn't strike me in the first review :D

Reply:
No worries :D best I could think of was `require_confirmations` or so. But I ended up using the one you suggested because it contains more information.